src/transformers/pipelines/conversational.py

```diff
@@ -277,11 +254,6 @@ def __call__(
             elif self.framework == "tf":
                 input_length = tf.shape(inputs["input_ids"])[-1].numpy()
 
-            if input_length > 0.9 * max_length:
```

Comment:
This was wrong because `max_length` corresponds to desired output length in tokens, whereas this warning concerns 
the input length. In the decoder-only setting it does not matter as much but it does in the encoder-decoder setting.

Reply:
I think this statement makes sense for DialoGPT but not for Blenderbot, so I'm happy to delete it